src/components/LandingFilter.jsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-env jquery */
+import React, { useState, useEffect } from 'react';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import {
+  Dropdown, Grid, Form
+} from 'semantic-ui-react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import theme from '../config/theme';
+import { regionOptions, paxOptions, ratingOptions } from '../config/constant';
+
+const GoButton = styled(Link)`
+  font-size: 50px;
```

Comment:
@OoiYueYing @denzelchung Lets refactor the files.

For each component, we should create 1 folder with 2 files inside.
For eg here, we can have a folder titled: LandingFilter. Inside LandingFilter folder, we can have LandingFilter.jsx and LandingFilter.styles.jsx to separate the code for logic and style.